### PR TITLE
ver final

### PR DESCRIPTION
**Pull request by aashvr**

### CODE REVIEW

**Review by rabbiasajjad-devsinc — COMMENTED** [review]
Explore Python enums for global variables of valid_city, valid_options

File: test.py
Comment (rabbiasajjad-devsinc) [inline test.py:1]: Make separate functions to check input validity instead of using if statements

Comment (aashvr) [reply test.py:1]: will resolve both issues. Thankyou.

File: utils.py
Comment (rabbiasajjad-devsinc) [inline utils.py:1]: Consider making separate functions for data processing and displaying results for better code modularity

**Review by rabbiasajjad-devsinc — COMMENTED** [review]
Add a proper readme file. You can use https://readme.so/ for it

File: test.py
Comment (aashvr) [reply test.py:1]: Resolved

File: utils.py
Comment (aashvr) [reply utils.py:1]: resolved!

**Comment by aashvr** [comment]
> Add a proper readme file. You can use https://readme.so/ for it

**Review by rabbiasajjad-devsinc — APPROVED** [review]

**Pull request merged**